_posts/en/newsletters/2024-10-04-newsletter.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'Bitcoin Optech Newsletter #323'
+permalink: /en/newsletters/2024/10/04/
+name: 2024-10-04-newsletter
+slug: 2024-10-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+*No significant news this week was found in any of our [sources][optech
+sources].*
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 1.0.0-beta.4][] is a release candidate of this library for
```

Comment:
[1.0.0-beta.5](https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.5) was published in the meantime.